src/main/cpp/t04_both2.cpp

```diff
@@ -23,9 +23,32 @@
 
 #include "t04_both2.h"
 #include <iostream>
+#include <set>
 
 using namespace std;
 
 int t04_both2() {
-
+    set<int> s1;
+    int n;
+    cin >> n;
+    for (int i = 0; i < n; i++) {
+        int x1;
+        cin >> x1;
+        s1.insert(x1);
+    }
+    set<int> s2;
+    set<int> s3;
+    cin >> n;
+    int k = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'k' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4591637)